pages/docs/quickstart.mdx

```diff
@@ -145,7 +145,7 @@ An approving review from one of LinkFree's maintainers will show a green check m
 ![maintainer review](https://user-images.githubusercontent.com/35927536/217060602-ba3678d1-f655-4805-a47d-e40f1ddcd59f.png)
 
 18. You will receive a GitHub notification when you have a comment, review or your Pull Request has been merged
```

Comment:
```suggestion

18. You will receive a GitHub notification when you have a comment, review, or your Pull Request has been merged. 
```

Reply:
No other bullet points end in a `.`, so adding one here wouldn't be consistent